server/src/main/java/com/service/CatBreedClassficationService.java

```diff
@@ -0,0 +1,59 @@
+package com.service;
+
+import com.alibaba.fastjson.JSONObject;
+import com.entity.Animal;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.File;
+import java.io.IOException;
+
+@Slf4j
+@Service
+public class CatBreedClassficationService {
+    @Autowired
+    private RestTemplate restTemplate;
+
+    private final String filePath= "Todo";
+
+    public void transferFile(MultipartFile file) throws IOException {
+        File file1=new File(filePath);
+        if(!file1.exists()){
+            boolean isSuccess=file1.mkdirs();
+            if(!isSuccess){
+                log.error("mkdirs failed");
+            }
+        }
+
+        String filename=file.getOriginalFilename();
+        String newFilePath=filePath+filename;
+        file.transferTo(new File(newFilePath));
+    }
+
+    public Animal getCatInfo(MultipartFile file) {
```

Comment:
Once again, we receive img from client, store it in memory, and classification network uses it directly(fetching from memory) and returns a target result we want.